### PR TITLE
Update supporting docs for new contributors

### PR DESCRIPTION
**Pull request by commjoen**
Thank you for submitting a pull request to the WrongSecrets-binaries repository!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -28,6 +32,12 @@ The minimum requirements for code contributions are:
 
 1. The code _must_ be compliant with the configured best practices for the language.
 2. All new and changed code _should_ have a corresponding unit and/or integration test.
+3. Any new binary challenge needs to adhere to the following criteria:
+- quickbuild.sh needs to be extended to build the challenge for various languages
+- a spoil command needs to be implemented that returns the answer
+- correct answers should print `This is correct! Congrats!\n`
+- incorrect answers should print at least `This is incorrect. Try again\n`.
+- Binaries should be generated in github actions/
```
Comment (bendehaan) [inline CONTRIBUTING.md:40]:
```suggestion
- Binaries should be generated in github actions
```

**Review by bendehaan — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+<!---
+Optionally, provide any helpful references that may help the reviewer(s).
+--->
+
+### Checklist:
+
+-   [ ] All the contributions made are solely the work of me and my co-authors
+-   [ ] I tested the changes in this PR (if applicable)
+-   [ ] I added unit tests to ensure my change works (when change in Java or on front-end code)
```
Comment (RemakingEden) [inline .github/PULL_REQUEST_TEMPLATE.md:38]: There is no reference to updating the UI tests if the changes are UI based. Should we add this in?

File: README.md [hunk 3]
```diff
@@ -7,6 +7,17 @@ Want to fix something you found in one of the binaries: open a ticket or a PR he
...
+### Contributors:
+
+Leaders:
+
+-   [Ben de Haan @bendehaan](https://github.com/bendehaan)
+-   [Jeroen Willemsen @commjoen](https://github.com/commjoen)
+
+Top contributors:
+
+-   [Joss Sparkes @remakingeden](https://github.com/remakingeden)
```
Comment (RemakingEden) [inline README.md:19]: I approve this change 🤝

**Comment by RemakingEden** [comment]
@commjoen I'm aware this has been merged but I left a comment above about a UI test checkbox in the PR template. If you agree I can make a PR to change.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (commjoen) [reply .github/PULL_REQUEST_TEMPLATE.md:38]: Nope: there is no ui in binaries

**Comment by RemakingEden** [comment]
@commjoen I was trying to sneakily delete my comments & review when I realised which repo I was in. To slow 🤦

**Pull request merged**